panel/src/components/Forms/Field/EntriesField.vue

```diff
@@ -229,6 +229,11 @@ export default {
 	watch: {
 		value: {
 			handler(entries) {
+				if (Array.isArray(entries) === false) {
+					this.entries = [];
+					return;
+				}
+
```

Comment:
```suggestion
				entries ??= [];

```

`Suggestion:` Shouldn't this be enough? Then we could still benefit from the `entries === this.values` check?